### PR TITLE
Fix simple example doesn't run on CodeSandbox

### PR DESCRIPTION
**Pull request by ValentinH**
The goal is to fix the [Codesandbox](https://codesandbox.io/s/github/marmelab/react-admin/tree/master/examples/simple) for the Simple example which is currently broken.

Here's an example codesandbox with the fix implemented in this PR: https://codesandbox.io/s/funny-colden-7p1in

### CODE REVIEW

File: examples/simple/package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
         "ra-input-rich-text": "^3.9.0",
         "ra-language-english": "^3.9.0",
         "ra-language-french": "^3.9.0",
+        "ra-ui-materialui": "^3.9.0",
```
Comment (djhi) [inline examples/simple/package.json:45]: `ra-ui-materialui` is a dependency of `react-admin`, this shouldn't be necessary

**Review by djhi — CHANGES_REQUESTED** [review]
I think we should update all the other react-admin dependencies to target `3.12.4` instead

File: examples/simple/package.json (hunk 1 above)
Comment (ValentinH) [reply examples/simple/package.json:45]: Yes that's what I thought as well. I'll try to update all the other dependencies then .

**Comment by fzaninotto** [comment]
I think the problem is in the CodeSandbox side; If you clone the defautl react-admin codesandbox, it works...

**Comment by ValentinH** [comment]
I don't really know what was wrong but after updating all ra-related packages, it works fine. I've updated my PR.

**Review by fzaninotto — APPROVED** [review]

**Comment by fzaninotto** [comment]
Except some of the packages don't have a version 3.12...

**Comment by ValentinH** [comment]
Indeed 🙈 
I fixed that by putting all the latest versions.

**Comment by fzaninotto** [comment]
your JSON is invalid. I suggest you test your patch before pushing it.

**Comment by ValentinH** [comment]
Sorry for the inconvenience. 😞

**Review by fzaninotto — APPROVED** [review]

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**